Firmware/Chameleon-Mini/Application/MifareClassic.c

```diff
@@ -74,9 +78,11 @@
 
 /*
 Source: NXP: MF1S50YYX Product data sheet
-
+*/
+#define BYTE_SWAP(x)            (((uint8_t)(x)>>4)|((uint8_t)(x)<<4))
```

Comment:
Byteswap?  looks like Nibble swap

Reply:
😆 it's sad but it looks like some void lines deletion and code moves make it seems like I changed waaaaay more than I did. This bit was already there, it's in RevG codebase as well, it is used for access control - that we do not implement yet.

Reply:
yeah,  I saw it later down.  I still don't agree with Chameleon code styles and naming conventions.